docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -1,14 +1,65 @@
 # Development Environment
 
-The development environment setup guide is for setting up your local development environment for JavaScript development; creating plugins and tools for extending WordPress and the block editor.
+This guide is for setting up your local environment for JavaScript development for creating plugins and tools to extend WordPress and the block editor.
 
-A development environment is a catch-all term for the setup on your computer to work. The three main pieces needed for our development environment are:
+A development environment is a catch-all term for what you need setup on your computer to work. The three main pieces needed for our development environment are:
 
 1. Node/NPM Development Tools
 2. WordPress Development Site
 3. Code Editor
 
-## Development Tools
+## Quickstart
+
+For experienced developers, here is a summary of the guide, see each section for more details and explanations.
+
+**1. Install Node development tools**
+
+Download and install [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm)
+
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+Quit and restart terminal
+```
+nvm install --lts
+```
+
+**2. WordPress Development Site**
+
+First download, install, and start [Docker Desktop](https://www.docker.com/products/docker-desktop) follows instructions on their site for your OS.
+
+- Install WordPress environment tool
+
+```
+npm -g install @wordpress/env
+```
+
+Open a text editor and create a `.wp-env.json` file with the following content:
+
+```json
+{
+	"core": null
+}
+```
```

Comment:
Btw the below comment showing my run through shows my tests saving the json file.

Remove the word json from the above code. So it becomes:

```
{
	"core": null
}
```

As the word json in the file gave this error:
```
paaljoachimromdahl@Paals-MacBook-Pro ~ % wp-env start
✖ Invalid .wp-env.json: Unexpected token j in JSON at position 0
```

Reply:
I'll be removing this step once confirmed.